modules/core/src/main/java/org/apache/ignite/internal/sql/SyncResultSetAdapter.java

```diff
@@ -32,7 +32,7 @@
 /**
  * Synchronous wrapper over {@link org.apache.ignite.sql.async.AsyncResultSet}.
  */
-class SyncResultSetAdapter<T> implements ResultSet<T> {
+public class SyncResultSetAdapter<T> implements ResultSet<T> {
```

Comment:
Do we really need to change visibility to public?

Reply:
The class requires also for test.I see two options here:
1. It can be public here
2  Move to utils package, but will be public in any case

Do you have another options?

Reply:
Let's leave it public then.